app/config.py

```diff
@@ -5,7 +5,7 @@
 FLASK_ENV = os.environ.get("FLASK_ENV")
 
 # CURRENTLY TEST API KEY IS BEING USED LOCALLY & ON CF.
-API_KEY = os.environ.get("GOV_NOTIFY_API_KEY") or os.environ.get(
+GOV_NOTIFY_API_KEY = os.environ.get("GOV_NOTIFY_API_KEY") or os.environ.get(
```

Comment:
Should this file not be removed? Isn't config coming from /config/default.py now?

Reply:
Also on the topic see note on create_app factory